README.md

```diff
@@ -66,6 +66,19 @@ struct MyMetric {
 }
 ```
 
+Timestamps are optional and can be set via the `timestamp` attribute, if not set the current time will be used:
```

Comment:
> if not set the current time will be used

I see this in the documentation added in a couple spaces. I think it would be more accurate to say that it will allow the Telegraf daemon running to set the time. This kind of implies we are setting the time, which we are not (in the case it is omitted).